lte/gateway/python/integ_tests/federated_tests/README.md

```diff
@@ -0,0 +1,95 @@
+---
+id: federated_integ_test
+title: Federated Integration Tests
+hide_title: true
+---
+# Federated Integration Tests (EXPERIMENTAL)
+
+The objective of Federated Integration Tests is to provide a test platform
+where to run Magma with all its components.
+
+As the diagram indicates, this test spins up AGW, FEG and Orc8r
+and uses S1aP and trf server to run tests.
+
+```mermaid
+graph LR;
+    S1AP --> AGW --> orc8r --> FEG --> 3ggp_CORE;
+    3ggp_CORE --> FEG --> orc8r --> AGW --> S1AP;
+    AGW --> trf_server;
+    trf_server --> AGW;
+```
+
+The processes will run on the following machines:
+- AGW will run on **magma** vm.
+- FEG and Orc8r will run on our **local docker**.
+- Traffic server will run on **magma_trf** vm
+- Test will be started from **magma_test** vm
+
+## Build environment
+### Automatic build
+On your host machine go to
+`cd  magma/lte/gateway/python/integ_tests/federated_tests` and run
+```
```

Comment:
```suggestion
```bash
```

Reply:
I think bash is the same as ```, but i added it just in case to all the boxes

Reply:
ohhh I was wondering if it adds any useful syntax coloring, but for bash I suppose that is minimal.